### ISSUE TITLE
Where can "v3" be obtained?

### ISSUE THREAD

**Issue by DanielSchetritt** [linuxserver/Heimdall#624]
Hello! I have noticed that several issues have been closed because the feature/bug in question has been implemented/fixed in v3 according to @KodeStar . Here are three such issues:
#386 #616  #610 

This is great news because I have been looking for solutions to some of these issues.

Now the question is where can we obtain the release of version 3? I use the docker container personally, but from looking at the git tags of this repository and of the docker-heimdall repo, I don't see any tags that begin with `3.` or `v3.` I also don't see any commits in the `master` branch that describe a version increase from 2 to 3. Nothing on the project website either.

I admit that I did not look for more than 15 minutes, so it is possible I might have missed something obvious.

Where has this been released? Could it be in another git repository hosting platform such as Gitlab?

Or is this still in development? If it's still in development, I'd be willing to contribute because I'd love to see these features get released. It looks like the latest commit pushed was 10 months ago. I can only speak for myself, but I would imagine that the open source community would be happy to help move things along!

Thank you, kindly. I eagerly await your reply.

**Comment by rajat-r** [comment]
Maybe https://github.com/linuxserver/heimdalljs ?

**Comment by Drallas** [comment]
Would be nice to have a bit more info about v3.0, what can we expect, will there be a beta version, how can we help?

**Comment by KodeStar** [comment]
heimdalljs was the correct answer, but the current plan is to try pulling some of the front end developed in that back into this. I have recently updated the underlying framework and so a new version should be out soon, once that is done I will start working on getting some of the front end moved across.
